src/auth/base.js

```diff
@@ -111,6 +111,7 @@ class Base {
             let clientOptions = this.parseAuthorization(header);
             if (clientOptions) {
                 searching = true;
+                clientOptions.host = req.headers["host"];
```

Comment:
You have `req` object in `findUser` function, makes sense to get `req.headers["host"]` there instead of changing clientOptions.